terraform/lambda.tf.json.template.py

```diff
@@ -0,0 +1,16 @@
+from azul import config
+from azul.deployment import emit_tf
+
+emit_tf({
+    "resource": [
+        {
+            "aws_lambda_layer_version": {
+                "dependency_layer": {
+                    "layer_name": config.qualified_resource_name("dependency_layer"),
```

Comment:
```suggestion
                    "layer_name": config.qualified_resource_name("dependencies"),
```

It's generally not necessary to encode the resource type in the resource name since each type defines its own isolated space for resource names.